curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md

```diff
@@ -6,12 +6,14 @@ challengeType: 5
 
 ## Description
 <section id='description'>
-<p>Implement a function that takes a string of four digits as its argument, with each digit from 1 ──► 9 (inclusive) with repetitions allowed, and returns an arithmetic expression that evaluates to the number 24. If no such solution exists, return "no solution exists."</p>
-<p>Rules:</p>
- Only the following operators/functions are allowed: multiplication, division, addition, subtraction
- Division should use floating point or rational arithmetic, etc, to preserve remainders.
- Forming multiple digit numbers from the supplied digits is disallowed. (So an answer of 12+12 when given 1, 2, 2, and 1 is wrong).
- The order of the digits when given does not have to be preserved.
+Implement a function that takes a string of four digits as its argument, with each digit from 1 ──► 9 (inclusive) with repetitions allowed, and returns an arithmetic expression that evaluates to the number 24.
+Rules:
+<ul>
+  <li>Only the following operators/functions are allowed: multiplication, division, addition, subtraction</li>
```

Comment:
@scissorsneedfoodtoo Should we keep the indentations here? Does the `<pre>` section problem apply here?